src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitService.java

```diff
@@ -161,6 +173,22 @@ public Page<FaultResult> getFaultResultsPage(UUID resultUuid, FaultResultsMode m
         return null;
     }
 
+    public Page<FeederResult> getFeederResultsPage(UUID resultUuid, List<Filter> filters, Pageable pageable) {
```

Comment:
transactional ?

Reply:
It's not used in this class yet, no transactional for getFaultResultsPage for instance

Reply:
It shouldn't have a visible effect on performance. When you load multiple things from the database, it's good practice to do it in a transaction to have a coherent view of the database. It should have been done in getFaultResultsPage, but we can fix it in a separate pr (note: since we have OSIV it doesn't matter anyway for now)

Reply:
OK then I had it for both in this PR if it's OK, anyway I edited the getFaultResultsPage method